src/components/Notification/BasicNotification.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { forwardRef, useState, useEffect } from 'react';
+import Snackbar from '@mui/material/Snackbar';
+import MuiAlert from '@mui/material/Alert';
+
+const Alert = forwardRef((props, ref) => (
+  <MuiAlert elevation={6} ref={ref} variant="filled" {...props} />
+));
+
+const BasicNotification = ({ severity, message }) => {
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    if (message !== '') {
+      setOpen(true);
+    }
+  }, [message]);
+
+  return (
+    <Snackbar open={open}>
```

Comment:
I think if you can add the slide transition on this (either the `up` or `left`) it will look super good...
https://mui.com/material-ui/react-snackbar/#change-transition